ethcore/src/engines/tendermint/mod.rs

```diff
@@ -401,8 +400,9 @@ impl Engine for Tendermint {
 		]
 	}
 
-	fn schedule(&self, _env_info: &EnvInfo) -> Schedule {
-		Schedule::new_post_eip150(usize::max_value(), true, true, true)
+	fn schedule(&self, block_number: BlockNumber) -> Schedule {
+		let eip86 = block_number >= self.params.eip98_transition;
```

Comment:
ugh, I'm going to replace it with `Schedule::from_params` for DRY after #4757 is merged. https://github.com/ethcore/parity/pull/4757/files#diff-ef2687136b4e032043a492e9e53dcb7cR177